portfolio.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Portfolio</title>
+    <link type="text/css" rel="stylesheet" href="normalize.css">
+    <link type="text/css" rel="stylesheet" href="styles.css">
+</head>
+
+<!--[if lte IE 9]><body class="ieLegacy"><![endif]-->
+<!--[if !IE]><body><![endif]-->
+
+<body>
+    <div>
+        <header class="container">
+            <nav>
+                <ul>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#">Portfolio</a></li>
+                </ul>
+            </nav>
+
+            <h1>My Portfolio</h1>
+        </header>
+
+        <main>
+            <div class="col3 col2">
```

Comment:
identify with col is happening based on size eg: col3 and col2-sm

Reply:
Changed .col2 to .col2Sm, does that suffice?